mln/models/static.py

```diff
@@ -26,6 +26,7 @@ class MLNError(Exception):
 	OUT_OF_VOTES = 46313
 	MLN_OFFLINE = 47570
 	MEMBER_NOT_FOUND = 52256
+	I_DONT_GET_IT = 46222

```

Comment:
These errors are actually a separate category in MLN's data xml, and the "I don't get it" message is not one of them. Just add the ID as a separate constant instead.